app/build.gradle

```diff
@@ -59,6 +59,9 @@ dependencies {
     debugImplementation "androidx.compose.ui:ui-tooling:$compose_ui_version"
     debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_ui_version"
 
+    // Compose Extended Icons
```

Comment:
Duplicated due to it has been added a few lines above.

Reply:
Corrected.

Reply:
You can click the "Resolve conversation" then!